containerfiles/initrd-builder/osbuilder/dracut/dracut-kata-nvidia-gpu-cc.conf

```diff
@@ -0,0 +1,31 @@
+# NVIDIA drivers
+add_drivers+=" nvidia-drm nvidia-modeset nvidia-uvm nvidia "
+add_drivers+=" efi_secret "
+
+# SNP
+add_drivers+=" sev-guest "
+# TDX
+add_drivers+=" tdx-guest tsm "
+
+# Dependencies from kata nvidia_rootfs.sh:chisseled_compute()
```

Comment:
Regarding the previous comment about leveraging dracut.conf.d, this NVIDIA configuration is what I am concerned about. The TODO will add to this (shared) list of entries, so it would be nice to have all that in a single file irrespective of CC or non-CC.